rwengine/src/objects/VehicleObject.cpp

```diff
@@ -385,6 +385,18 @@ void VehicleObject::tickPhysics(float dt) {
             brakeF = 2.f * std::min(1.f + kM, 4.f);
         }
 
+        if (isStopped() && std::abs(throttle) < 0.1f) {
+            btVector3 v = collision->getBulletBody()->getLinearVelocity();
+            v.setX(0.f);
+
+            collision->getBulletBody()->setLinearVelocity(v);
+
+            for (int w = 0; w < physVehicle->getNumWheels(); ++w) {
+                btWheelInfo& wi = physVehicle->getWheelInfo(w);
+                wi.m_rotation = 0.f;
```

Comment:
The problem with setting the wheel angle to zero is, that it visibly snaps into that position. During the transition from Moving to Stopping, the current wheel angle should be kept.

Reply:
Ah right, I hadn't thought of/witnessed that.